lib/p2p/Peer.ts

```diff
@@ -190,6 +192,21 @@ class Peer extends EventEmitter {
 
     // TODO: Check that the peer's version is compatible with ours
 
+    // request peer's known nodes only if p2p.discover option is true
+    if (this.discover) {
+      if (this.discoverminutes === 0) {
+        // timer is disabled
+        this.sendPacket(new packets.GetNodesPacket());
```

Comment:
you can take this out of the `if` clause - it's executed in both cases.
so there should only one `if` statement - checking whether `this.discoverminutes` is truthy.

Reply:
right